### DISCUSSION TITLE
Welcome to github-error-solve Discussions!

### DISCUSSION THREAD

**Discussion by devvsakib** [devvsakib/github-error-solve#90] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by abhiraj-ku** [comment]
Hey I am facing this issue (screenshot attached) and i have tried implementing solutions from stackoverflow but it didn't work out ..
so please suggest me the solution of this 

Issue: There isn't anything to compare

![IMG_20230217_222642](https://user-images.githubusercontent.com/113116498/219932588-da522a18-a8bf-4bfb-9db2-f6f75774570f.jpg)

**Reply by devvsakib** [reply]
did your push files from your pc?
